apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -678,10 +576,10 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
               }}
               useRebrandedLikeStyles
             >
-              <button key={0} type="button" value={Toggle.EFFECT}>
+              <button type="button" value={Toggle.EFFECT}>
```

Comment:
keys are only meaningful in a loop. They're not needed inside of static content like this, so I dropped them.